docs/advanced.rst

```diff
@@ -1178,53 +1178,82 @@ If the default exception conversion policy described
 is insufficient, pybind11 also provides support for registering custom
 exception translators.
 
-The function ``register_exception_translator(translator)`` takes a stateless
-callable (e.g. a function pointer or a lambda function without captured
-variables) with the following call signature: ``void(std::exception_ptr)``.
+For declaring a simple, single exception translation that translates a C++
+exception into a new Python exception using the C++ exception's ``what()``
+method, a helper function is available:
+
+.. code-block:: cpp
+
+    py::translate_exception<CppExceptionName>(module, "PythonExceptionName");
+
+This call creates a PythonExceptionName exception object in the given module
+and automatically translates any CppExceptionName exceptions that pybind11
+encounters into PythonExceptionName instances to be handled by the calling
+Python code.
+
+When more advanced exception translation is needed, the function
+``py::register_exception_translator(translator)`` can be used to perform custom
```

Comment:
[..] to register functions that can translate arbitrary exceptions types (and which may include additional logic to do so). The function takes [...]